rules/python/shared/lang/dynamic_input.yml

```diff
@@ -33,6 +38,10 @@ auxiliary:
                 values: [argparse]
               - variable: NAME
                 values: [ArgumentParser]
+  - id: python_shared_lang_dynamic_input_input
```

Comment:
Will this match keyword params also?

Reply:
Ahh, thanks, good catch! Added an extra pattern (and test): https://github.com/Bearer/bearer-rules/pull/450/commits/d621942103875ae11dc45db071d39ef457b5baf7